app/controllers/rents_controller.rb

```diff
@@ -27,6 +28,10 @@ def send_mails(rent)
     RentNotificationWorker.perform_in((rent.to - rent.from).days, rent.id)
   end
 
+  def translate(rent)
```

Comment:
I'm pretty sure we will want to set the locale for the request always, and not only on RentsController#create. What do you think about moving this to the ApiController and executing it always with a `before_action`? Remember you can access the [`current_user`](https://maicolben.gitbooks.io/devise-token-auth/content/docs/usage/controller_methods.html) (if the user is logged in)